hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/handlers/ContractCallLocalHandler.java

```diff
@@ -142,6 +146,7 @@ public Response findResponse(@NonNull final QueryContext context, @NonNull final
         requireNonNull(header);
 
         final var component = provider.get().create(context, instantSource.instant(), CONTRACT_CALL_LOCAL);
```

Comment:
To be consistent, should there be a getTransactionComponent in the abstract class for this as well?

Reply:
ContractCallLocal is a bit different than the other uses - it's our only query tx.  The abstract base class is in `app.spi.workflows`.  I could introduce an intermediate abstract class that we'd own ... but even then, we're not likely to get another query tx like call local so it would be kind of a one-off.

I'll introduce a named method to hold this rather specific line.